src/rpc/blockchain.cpp

```diff
@@ -33,6 +34,7 @@
 
 #include <univalue.h>
 
+#include <boost/algorithm/string.hpp>
```

Comment:
Hey -- I was just browsing around the extant PRs -- quick question: why include this header? I don't see any new lines using it..  does it fix a compile error that was already there or something?

Reply:
Looks like you're right, it is not needed. It was in the original PR. Maybe the original author used it in an earlier revision and forgot to take it out.

I will remove it if I do a revision on this PR.

Reply:
Yeah this happens to me a lot -- include some header for something -- take that something out -- leave header in. So I am vigilant always to detect that since it's a bane.. :)